charts/folding-at-home/v0.1.0/templates/deployment.yaml

```diff
@@ -0,0 +1,151 @@
+            {{- end}}
+
+          resources:
+            {{- if .Values.gpu }}
+            limits:
+              nvidia.com/gpu: "1" # How many gpu's you want to donate per ndoe
+              cpu: {{ .Values.limits.cpu | quote }} # How much CPU you wish to donate per node
+              memory: {{ .Values.limits.memory | quote }} 
+            requests:
+              nvidia.com/gpu: "1"
+              cpu: {{ .Values.requests.cpu | quote }} 
+              memory: {{ .Values.limits.memory | quote }}       
+            {{- else }}
+            limits:
+              cpu: {{ .Values.limits.cpu | quote }} # How much CPU you wish to donate per node
+              memory: {{ .Values.limits.memory | quote }} 
+            requests:
+              cpu: {{ .Values.requests.cpu | quote }} 
+              memory: {{ .Values.limits.memory | quote }} 
+            {{- end}}
+
+          # Make the container harder to break out of or exploit
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 1234
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+          volumeMounts:
+            - mountPath: /var/lib/fahclient
+              name: fahclient
+      # We make an emptyDir to mount on the work directory /var/lib/fahclient
+      # so we can make the rest of the container's root filesystem read-only
+      volumes:
+        - name: fahclient
+          emptyDir: {}
+        # Uncomment if using ConfigMap config.xml
+        # - name: fah-worker-config
+        #   configMap:
+        #     name: foldingathome-config
+      initContainers:
+        - name: copy-config
+          imagePullPolicy: Always
+          image: "richstokes20/fah-covid:latest"
+          command:
+            - "sh"
+            - "-c"
+            - "cp /etc/fahclient/config.xml /var/lib/fahclient/config.xml"
+            # Use this cp command instead if using ConfigMap config.xml
+            # - "cp /etc/fahclient-config/config.xml /var/lib/fahclient/config.xml"
+          securityContext:
+            runAsNonRoot: true
+            runAsUser: 1234
+            readOnlyRootFilesystem: true
+            allowPrivilegeEscalation: false
+          volumeMounts:
+            - mountPath: /var/lib/fahclient
+              name: fahclient
+            # Uncomment if using ConfigMap config.xml
+            # - name: fah-worker-config
+            #   mountPath: /etc/fahclient-config
+      # priorityClassName: low-priority-class
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: foldingathome-config
+data:
+  config.xml: |
+    <config>
+      <!--
+        To set your user name, team and passkey just edit the text
+        in quotes below.
+      -->
+
+      <!-- User Information -->
+      <user value=""/> <!-- Enter your user name here -->
+      <team value=""/>         <!-- Your team number -->
+      <passkey value=""/>       <!-- 32 hexadecimal characters if provided -->
```

Comment:
Nit: passkey is optional. Think perhaps should tweak the comment to make this more clear?
Do we need to explain what the passkey is/does? People shouldn't need it unless when they created their (also optional) user they opted into making one.
IIRC the passkey is essentially something you can opt into creating for your user so others can't claim your work (not sure how that could happen though).

Reply:
good point, i'll add that.